searchcore/src/vespa/searchcore/proton/server/lid_space_compaction_handler.cpp

```diff
@@ -93,7 +94,7 @@ LidSpaceCompactionHandler::handleCompactLidSpace(const CompactLidSpaceOperation
 {
     assert(_subDb.sub_db_id() == op.getSubDbId());
     _subDb.feed_view()->handleCompactLidSpace(op);
-    _subDb.feed_view()->forceCommit(op.getSerialNum(), std::move(compact_done_context));
+    _subDb.feed_view()->forceCommit(CommitParam(op.getSerialNum(), true), std::move(compact_done_context));
```

Comment:
Why are we using _forceUpdateStats=true_ here? That is not equal to the previous behaviour.

Reply:
Removed semantic change